src/utils/helpers.ts

```diff
@@ -252,7 +252,7 @@ export async function generateGitHubWebUrl(
   notification: Notification,
   accounts: AuthState,
 ): Promise<string> {
-  let url: string;
+  let url = notification.repository.html_url;
```

Comment:
Well, this is a bit of a messy situation, because the default case made more sense, but obviously it doesn't always enter that conditional... Nice bug find!